lswin.py

```diff
@@ -1,30 +1,141 @@
 #!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import sys
+from dataclasses import dataclass, fields
+from itertools import tee
+from optparse import OptionParser
+from typing import Iterable, Final
 
 import Quartz
 
-#wl = Quartz.CGWindowListCopyWindowInfo( Quartz.kCGWindowListOptionOnScreenOnly | Quartz.kCGWindowListExcludeDesktopElements, Quartz.kCGNullWindowID)
```

Comment:
@jjqq2013 what's this line meaning? if need, can add a command option for this. 😄